src/components/Class/index.js

```diff
@@ -765,6 +806,7 @@ function Class({ classToEdit, indicator }) {
                           checkBoxHandler(
                             e,
                             "MO",
+                            1,
```

Comment:
Yes. That abbreviations are being send to backend

Reply:
Lekin, kyaa back end sankhyaon se determine dinon ke naam nahi kar sakta?  I think it might be better if there were a constants file that contained a mapping from the day numbers to their abbreviations.  This constants file could then be shared by the front and back ends so if the abbreviations were to change (e.g., MO to MON), you'd only need to change them in one place.

Reply:
I don't know about backend